docs/getting-started/installing-service-binding.md

```diff
@@ -4,13 +4,46 @@ sidebar_position: 1
 
 # Installing Service Binding
 
-## Compatiblity Note
+The Service Binding Operator can be installed on the following version of Kubernetes and OpenShift.
 
-## Operator
+- Kubernetes version 1.20 or above.
+- OpenShift version 4.7 or above.
```

Comment:
we still release for OCP 4.6+ - please update accordingly kubernetes version. Check our CI job to see what kubernetes version we use for testing.

Reply:
Done. f2c70cce6c5ef069a1e10dea2e584fed54cd294b